tests/behaviour/steps/cli.py

```diff
@@ -80,6 +85,35 @@ def step_impl(context):
     assert log_file.exists(), f"Files in log directory: {list(log_dir.rglob('**/*'))}"
 
 
+@then("the command line output should display the log message")
+def step_impl(context):
+    (status_code, log_message) = context.csvcubed_cli_result
+    expected_log_message = context.text.strip()
+    assert expected_log_message in log_message, log_message
+
+
+@then("the command line output should not display the log message")
+def step_impl(context):
+    (status_code, log_message) = context.csvcubed_cli_result
+    expected_log_message = context.text.strip()
+    assert expected_log_message not in log_message, log_message
+
+
+@then(
```

Comment:
Is this the right location for this step definition since it isn't used in the CLI feature file?

Reply:
Moved it to the skoskodelist.py